core/src/main/kotlin/org/evomaster/core/problem/enterprise/EnterpriseIndividual.kt

```diff
@@ -343,6 +343,13 @@ abstract class EnterpriseIndividual(
         killChildren { it is SqlAction && actions.contains(it)}
     }
 
+    /***
+     * remove specified list of [HostnameResolutionAction] from the initializing actions.
+     */
+    fun removeHostnameResolutionAction(actions: List<HostnameResolutionAction>) {
+        killChildren {  it is HostnameResolutionAction && actions.contains(it) }
```

Comment:
as it is, it would not work properly. if checking for equality (ie in `contains`), need to make sure that `HostnameResolutionAction` overrides `equals` and `hasthTo` methods

Reply:
added `hashCode` and `equals`